src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1332,14 +1334,14 @@ public void LanguageVersionAdded_Canary()
             // - update the "UpgradeProject" codefixer
```

Comment:
Also, could you add a test for the `LanguageVersionExtensionsInternal.GetErrorCode` method. It should loop over `Enum.GetValues(typeof(LanguageVersion)).Cast<LanguageVersion>()` so that it breaks any time a new enum value is added.